README.md

```diff
@@ -1,65 +1,58 @@
+This repository contains three primary resources:
 
-### Class Material
+- Our current [on-demand courses](https://services.github.com/on-demand/) can be found in the [paths directory](/paths)
+- The translations of our popular [Git Cheat Sheets](https://services.github.com/on-demand/downloads/github-git-cheat-sheet.pdf) can be found in the [downloads directory](/downloads). We're always looking for more. _P.S._ Right now the PDF generation is a manual process so please mention @github/services-training for assistance in getting your translation moved to the PDF.
+- The recommended [Training Path](https://services.github.com/on-demand/path/) can be found [here](/self-study.md).
 
-[GitHub for Developers](https://services.github.com/kit/courses/github-for-developers.html) and [GitHub for Everyone](https://services.github.com/kit/courses/github-for-everyone.html) are the two class types taught by the GitHub Training team. Their content is composed of several modules listed in the [`_modules/`](https://github.com/github/training-kit/tree/master/_modules/) subdirectory.
+## Our Content Philosophy
 
-### Cheat Sheets
+We are eager to create materials that are easy to use and follow! To that end, here are a few guidelines we ask you to keep in mind:
 
-The Git and GitHub Cheat Sheets are located within the [`downloads/`](https://github.com/github/training-kit/tree/master/downloads/) directory. There are some translations available and we're always looking for more. If you are looking to get a `.pdf` generated for the html version of a cheatsheet, please start an issue and @mention [@brntbeer](https://github.com/brntbeer) or [@crichid](https://github.com/crichid) and they will see to the request.
+- We focus on providing clear, step-by-step instructions for completing an activity, giving the learner the opportunity to study the concept in greater detail with the Tell Me Why feature.
+- We focus on minimalism.
+- We use jekyll and [Markdown](https://guides.github.com/features/mastering-markdown/).
+- The theme for the on-demand training is [Minimal Mistakes](https://mmistakes.github.io/minimal-mistakes/) and has some [amazing documentation](https://mmistakes.github.io/minimal-mistakes/docs/quick-start-guide/).
 
-### Legacy Class Material
 
-This repository also contains our legacy course content. You can find these in the top-level directories:
-- [`foundations/`](https://github.com/github/training-kit/tree/master/foundations)
-- [`intermediate/`](https://github.com/github/training-kit/tree/master/intermediate)
-- [`advanced/`](https://github.com/github/training-kit/tree/master/advanced)
+## Building and Packaging
 
+#### Building and Testing Locally
 
-## File Format
+When you are ready to test your changes, you will want to build the repository locally. This is fully automated through a series of shell scripts based [the scripts to rule them all](https://github.com/github/scripts-to-rule-them-all)!
```

Comment:
Should we include that Ruby and homebrew are necessary for this process? (Are they?)